src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/LanguageServices/Handlers/AbstractEvaluationCommandLineHandler.cs

```diff
@@ -183,10 +183,7 @@ private void ApplyChangesToContext(IWorkspaceProjectContext context, IProjectCha
 
                 foreach (string includePath in difference.ChangedItems)
                 {
-                    UpdateInContextIfPresent(context, includePath, previousMetadata, currentMetadata, isActiveContext, logger);
-
-                    // TODO: Check for changes in the metadata indicating if we should ignore the file
-                    // in the current configuration.
+                    HandleEvaluationMetadataChange(context, includePath, previousMetadata, currentMetadata, isActiveContext, logger);
```

Comment:
Note that `UpdateInContextIfPresent` has been renamed to `HandleEvaluationMetadataChange` to reflect that is does more than update already-present items.